README.md

```diff
@@ -87,16 +87,36 @@ $ xcode-select --install
 </details>
 
 <details>
-<summary>Installing PhantomJS</summary>
+<summary>Browser testing</summary>
 
-We use the [Poltergeist JS driver](https://github.com/teampoltergeist/poltergeist)
-for Capybara tests, which requires PhantomJS. Install this with your favourite
-package manager, e.g.:
+We use [headless chrome](https://developers.google.com/web/updates/2017/04/headless-chrome)
+for Capybara tests, which require JavaScript. You will need to install Chrome >= 59. 
+Where we don't require JavaScript to test a feature we use Capybara's default driver 
+[RackTest](https://github.com/teamcapybara/capybara#racktest) which is ruby based 
+and much faster as it does not require a server to be started.
+
+**Debugging:**
+
+To debug a spec that requires JavaScript, you need to set a environment variable called CHROME_DEBUG.
+It can be set to any value you like.
+
+Examples:
+
+```
+$ CHROME_DEBUG=1 bundle exec rspec
+```
```

Comment:
I say cut out the other examples here and just leave this one. It's not really adding anything but noise ... there's no value in understanding what you can set `CHROME_DEBUG` to.